<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserEditor</class>
 <widget class="QWidget" name="UserEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>168</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="username_label">
         <property name="text">
          <string>Username:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="old_pwd_label">
         <property name="text">
          <string>Old password:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="new_pwd_label">
         <property name="text">
          <string>New password:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="repeat_new_pwd_label">
         <property name="text">
          <string>Repeat new password:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="username"/>
       </item>
       <item>
        <widget class="QLineEdit" name="old_pwd"/>
       </item>
       <item>
        <widget class="QLineEdit" name="new_pwd"/>
       </item>
       <item>
        <widget class="QLineEdit" name="repeat_new_pwd"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="vspacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
